Source/r_draw.c

```diff
@@ -270,6 +271,133 @@ void R_DrawTLColumn (void)
   }
 } 
 
+//
```

Comment:
Please quote the sources here, e.g. `eternity-engine/source/r_draw.c:L200-400`.

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/483/commits/ee455c291efa9afb2d6fb3b5e3f757e87a001de5